src/store/wallet/wallet-reducers.ts

```diff
@@ -51,6 +52,18 @@ export const walletReducers = createReducer<WalletState>(walletInitialState, bui
       isVisible
     }))
   );
+  builder.addCase(loadWhitelistAction.success, (state, { payload: tokensMetadata }) =>
+    updateCurrentAccountState(state, account => ({
+      ...account,
+      tokensList: account.tokensList.concat(
+        tokensMetadata.map(token => ({
+          slug: getTokenSlug(token),
+          balance: '0',
+          visibility: VisibilityEnum.InitiallyHidden
```

Comment:
I saw some problems:  
1. all tokens will be hidden for empty accounts  
2. visibility will be overridden for tokens, after the app restart 


I think we need to keep some tokens visible (from the local whitelist)